README-docker-compose.md

```diff
@@ -229,6 +187,79 @@ Ubuntu: Skip install of docker-sync, fswatch, and unison. instead...
 
 ## Application Debugging
 
+### Debugging Services
+
+The OSF is supported by several services which function independently from the main site and need some configuration to be modified using docker-sync. If you don't need to make changes to Waterbutler, MFR etc. you can ignore this.
+
+  Uncomment the appropriate code in docker-compose.override.yml and docker-sync.yml for your desired container and be sure to specify the relative path to your service code directories.
+  This makes it so your local changes will be reflected in the docker containers. Until you do this none of your changes will have any effect.
+  For example if you wanted to the modify Waterbutler you would uncomment the following.
+  
+  - In `docker-compose.override.yml`:
+
+    ```yml
+    services:
+      wb:
+        volumes_from:
+          - container:wb-sync
+
+    ...
+    ```
+
+  - In `docker-sync.yml`:
+
+    ```yml
+    syncs:
+      wb-sync:
+        src: '../waterbutler'
+        dest: '/code'
+        sync_strategy: 'unison'
+        sync_excludes_type: 'Name'
+        sync_excludes: ['.DS_Store', '*.pyc', '*.tmp', '.git', '.idea']
+        watch_excludes: ['.*\.DS_Store', '.*\.pyc', '.*\.tmp', '.*/\.git', '.*/\.idea']
+
+    ...
+    ```
+  
+  Modifying these files will show up as changes in git. To avoid committing these files, run:
+  
+  ```bash
+  git update-index --skip-worktree docker-compose.override.yml docker-sync.yml
+  ```
+  
+  To be able to commit changes to these files again, run:
+  
+  ```bash
+  git update-index --no-skip-worktree docker-compose.override.yml docker-sync.yml
+  ```
+
+  This will allow local changes to be reflected in your Docker environment, but you'll have to run `docker-compose up --force-recreate <container name>` every time you modify code in order to see your changes reflected in the application. But fear not! Setting `DEBUG=1` and `SERVER_CONFIG_DEBUG=1` in `docker-compose.wb.env` or `docker-compose.mfr.env` will enable live reload for those services, so your changes will take effect automatically in a few seconds.
```

Comment:
Small note: `--force-recreate` is only required once, when you first changes the volumes associated with docker sync. After that it will always know to look for the correct storage volume when the docker container is "turned on".

I suggest the following wording, to place greater emphasis on the recommended solution:
> The first time that sync settings are changed, you will need to run `docker-compose up --force-recreate <container name>`. To see the effect of code changes as you work (without needing to restart the container), you will need to separately turn on debug mode for the service: ...